os2forms_webform_submission_log.services.yml

```diff
@@ -1,6 +1,16 @@
 services:
   Drupal\os2forms_webform_submission_log\WebformSubmissionLogLogger:
+    arguments:
+      - '@os2forms_webform_submission_log.mailer'
     tags:
       - { name: logger }
 
+  os2forms_webform_submission_log.mailer:
+    class: Drupal\os2forms_webform_submission_log\WebformSubmissionLogMailer
```

Comment:
Use the fully qualified name to defined the service
(cf. `Drupal\os2forms_webform_submission_log\WebformSubmissionLogLogger` and https://www.drupal.org/docs/drupal-apis/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8#s-defining-services-by-fully-qualified-name-of-php-namespace).

Reply:
Fixed